Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -438,6 +438,18 @@ define([
         return tile._screenSpaceError > baseScreenSpaceError;
     }
 
+    function canTraverse(tileset, tile) {
+        if (tile.children.length === 0) {
+            return false;
```

Comment:
So the consensus from https://github.com/AnalyticalGraphicsInc/cesium/issues/7007 is not to throw a warning, correct?

It seems like it would be fairly trivial to include here with a one-time warning.